<launch>
    <arg name="image_topic" default="/seek_thermal/image_rect_color"/>
    <arg name="camera_info_topic" default="/seek_thermal/camera_info"/>
    <arg name="map_frame" default="map"/>
    <arg name="use_rviz" default="true"/>

    <node pkg="thermal_88" exec="thermal_wrapper_node" name="thermal_pipeline" output="screen">
        <param name="image_topic" value="$(var image_topic)"/>
        <param name="camera_info_topic" value="$(var camera_info_topic)"/>
        <param name="use_rviz" value="$(var use_rviz)"/>
    </node>

    <node exec="rviz2" name="calib_rviz" pkg="rviz2" args="-d $(find-pkg-share thermal_88)/config/pipeline.rviz" if="$(var use_rviz)"/>
</launch>